### PR TITLE
updated convertfrom-sid to include -21- SID's in main collection

### PR DESCRIPTION
**Pull request by puttyq**
love the library and have used parallel powershell all over the place. thought i'd see how to contribute. small fix and enhancement to add more well known SIDs

### CODE REVIEW

**Comment by RamblingCookieMonster** [comment]
Cool, good idea, thanks!

File: ConvertFrom-SID.ps1 [hunk 1]
```diff
@@ -123,10 +148,10 @@ function ConvertFrom-SID {
 
             #Display the results
             New-Object -TypeName PSObject -Property @{
-                SID = $id
+                SID = $id;
```
Comment (RamblingCookieMonster) [inline ConvertFrom-SID.ps1]: Hiyo!  Don't need the semi-colon here.  Going to test this _suggestion_ thingy!

```suggestion
                SID = $id
```

Comment (puttyq) [reply ConvertFrom-SID.ps1]: oops - sorry, was playing. i missed that before submitting

**Comment by RamblingCookieMonster** [comment]
Alrighty, going to merge this, and make a minor change (return original SID vs. the sanitized one) - thanks again!

**Pull request merged**